tests/python/pants_test/init/test_logging.py

```diff
@@ -1,71 +1,51 @@
 # coding=utf-8
-# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
We aren't supposed to update copyright year iirc.

Reply:
Reverted